package.json

```diff
@@ -164,7 +164,7 @@
   },
   "packageManager": "pnpm@9.6.0",
```

Comment:
pnpm >=9.2.0 removed the strictness so this should still remain permissive as long as a version matching that is used